services/publishing/Tweek.Publishing.Tests/Helpers/HooksHelperTests.cs

```diff
@@ -190,13 +197,15 @@ public class HooksHelperTests {
       A.CallTo(() => fakeGit(gitCommand)).Returns(gitOutput);
 
       // GetAllKeyHooks
-      var allHooks = new Hook[] {
+      var allHooks = new[] {
         new Hook("0", "a/b/c", "notification_webhook", "http://some-domain/awesome_hook"),
         new Hook("1", "a/b/c", "notification_webhook", "http://some-domain/another_awesome_hook"),
         new Hook("2", "a/b/*", "notification_webhook", "http://another-domain/ok_hook"),
         new Hook("3", "a/b/d", "notification_webhook", "http://some-domain/awesome_hook"),
         new Hook("4", "c/q/r", "notification_webhook", "http://fourth-domain/meh_hook"),
-        new Hook("5", "a/b/c", "not_a_notification_hook", "http://fifth-domain/should_not_be_called_hook")
+        new Hook("5", "a/b/c", "not_a_notification_hook", "http://fifth-domain/should_not_be_called_hook"),
```

Comment:
hmm would probably be clearer to also change this example hook name to `not_a_post_commit_hook`